packages/react-dom/src/__tests__/ReactDOMServerIntegrationElements-test.js

```diff
@@ -651,7 +651,7 @@ describe('ReactDOMServerIntegration', () => {
       });
     });
 
-    describe('component hierarchies', async function() {
```

Comment:
`describe` must be `void` (in reality it must be synchronous) in Jest. `async` here was always an error, but it throws with jest 24.

https://jestjs.io/docs/en/troubleshooting#defining-tests